capture.py

```diff
@@ -352,16 +352,30 @@ def apply_view(panel, **options):
         cmds.setAttr(attr, value)
 
 
-def parse_active_view():
-    """Parse the current settings from the active view"""
+def parse_active_panel():
```

Comment:
The name "parse" does feel somewhat misplaced because of it's simplicity in lines of code, but it fits the overall naming of the other functions. Otherwise we could name it `get_active_panel()`. Just as fitting I would say, but than going a bit rogue compared to the names of the other methods.

Whether we need the method and if we can avoid the complexity, maybe yes. Looking at the closed issues so far it seems there have been some "trouble" to get just a focused model panel without hickups:

https://github.com/abstractfactory/maya-capture/issues/41
https://github.com/abstractfactory/maya-capture/issues/39

Both related to just getting the panel from the current focus.